primitives/fee-market/src/lib.rs

```diff
@@ -213,6 +213,42 @@ where
 	}
 }
 
+/// The detail information about slash behavior
+#[derive(Clone, Encode, Decode, Eq, PartialEq, Debug)]
+pub struct SlashReport<AccountId, BlockNumber, Balance> {
+	pub lane: LaneId,
+	pub message: MessageNonce,
+	pub sent_time: BlockNumber,
+	pub confirm_time: Option<BlockNumber>,
+	pub delay_time: Option<BlockNumber>,
+	pub account_id: AccountId,
+	pub amount: Balance,
```

Comment:
After discussing with Yalin, both s2s message and fee market pallets will clean the old order/message data as time goes by. So it is better to keep all these fields.